src/components/controls/Controls.jsx

```
@@ -0,0 +1,54 @@
+import { useDispatch } from 'react-redux';
+import { TOGGLE_GAME_STARTED_LIST } from '../../stores/gameState/gameStateActions';
+import { useSelector } from 'react-redux';
+import GameControls from './GameControls';
+import { useFlag } from '@unleash/nextjs';
+
+function Controls() {
+  const allowMultiplayer = useFlag("multiplayer");
+  const [loading, setLoading] = useState(false);
+  const { start, join, sessionId } = useMultiplayer(allowMultiplayer);
+  const { gameStarted } = useSelector(state => state.gameState);
+  let dispatch = useDispatch();
```

ESLint

[error] 'dispatch' is never reassigned. Use 'const' instead.

(prefer-const)